vm/isolated/isolated.go

```diff
@@ -19,16 +19,20 @@ import (
 	"github.com/google/syzkaller/vm/vmimpl"
 )
 
+var pstoreConsoleFile string = "/sys/fs/pstore/console-ramoops-0"
+
 func init() {
 	vmimpl.Register("isolated", ctor, false)
 }
 
 type Config struct {
-	Host         string   `json:"host"`           // host ip addr
-	Targets      []string `json:"targets"`        // target machines: (hostname|ip)(:port)?
-	TargetDir    string   `json:"target_dir"`     // directory to copy/run on target
-	TargetReboot bool     `json:"target_reboot"`  // reboot target on repair
-	USBDevNums   []string `json:"usb_device_num"` // /sys/bus/usb/devices/
+	Host             string   `json:"host"`               // host ip addr
+	Targets          []string `json:"targets"`            // target machines: (hostname|ip)(:port)?
+	TargetDir        string   `json:"target_dir"`         // directory to copy/run on target
+	TargetReboot     bool     `json:"target_reboot"`      // reboot target on repair
+	USBDevNums       []string `json:"usb_device_num"`     // /sys/bus/usb/devices/
+	PostRepairScript string   `json:"post_repair_script"` // script to execute after each repair
```

Comment:
Repair is an internal term used solely in this vm impl as implementation detail. PostRepairScript is a confusing name for me as a user. "startup script" seems to be more widespread term, e.g.:
https://cloud.google.com/compute/docs/startupscript